### PR TITLE
Display hubs used for rep first

### PR DESCRIPTION
**Pull request by yattias**

### CODE REVIEW

File: components/Author/Tabs/FeedCard.tsx [hunk 1]
```diff
@@ -64,7 +64,7 @@ export type FeedCardProps = {
   created_by: any;
   document: any;
   created_date: any;
-  documentFilter: any;
+  documentFilterdocumentFilter: any;
```
Comment (gzurowski) [inline components/Author/Tabs/FeedCard.tsx:67]: Was this change intended?

File: components/Author/Tabs/FeedCard.tsx [hunk 2]
```diff
@@ -165,7 +165,12 @@ function FeedCard({
 
   const visibleDate = parsedDoc?.publishedDate || parsedDoc?.createdDate;
 
-  const parsedHubs = (hubs || []).map(parseHub);
+  // Sort hubs by displaying ones with isUsedForRep first.
+  // This approach should be deprecated once we introduce "hub" namespace.
```
Comment (gzurowski) [inline components/Author/Tabs/FeedCard.tsx:169]:
```suggestion
  // TODO: This approach should be deprecated once we introduce "hub" namespace.
```

**Review by gzurowski — APPROVED** [review]
LGTM 👍

File: components/Author/Tabs/FeedCard.tsx (hunk 1 above)
Comment (yattias) [reply components/Author/Tabs/FeedCard.tsx:67]: Good catch @gzurowski probably some keyboard error
